app/views/content/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme.md

```diff
@@ -19,38 +19,36 @@ keywords:
 ---
 One of the great benefits of a teaching career is a secure pension that will help you save for your future. 
 
-The teachers’ pension scheme is one of the most generous in the country. These are some of the key benefits of the scheme.
+The teachers’ pension scheme is one of the most generous in the country.
 
-## Easy to get started
+## How the teachers' pension works
 
-As a teacher, you’ll be automatically enrolled at the start of your career, and remain in the scheme when moving teaching jobs, unless you opt out. 
+You’ll automatically be enrolled into the pension scheme at the start of your career. You'll remain in the scheme when moving teaching jobs, unless you opt out. 
 
 Under the scheme, you'll automatically pay from your salary into your pension each month. At the same time, your employer also contributes.
 
 Once you join the scheme, it's quick and easy to manage your pension online.
 
-## Based on your salary
+## Salary-based pension contributions
 
 The teachers' pension scheme includes an employer contribution of 28.6%.
 
-It’s a ‘defined benefit’ pension, which means that it’s based on your salary rather than the amount you contribute.
+It’s a ‘defined benefit’ pension, which means it’s based on your salary rather than the amount you contribute.
```

Comment:
Do we need the quote marks (I know they are in prod already!)? Pet peeve!
```suggestion
It’s a defined benefit pension, which means it’s based on your salary rather than the amount you contribute.
```